common.mk

```diff
@@ -566,6 +568,10 @@ OPTEE_OS_COMMON_FLAGS ?= \
 	DEBUG=$(DEBUG) \
 	CFG_IN_TREE_EARLY_TAS="$(CFG_IN_TREE_EARLY_TAS)"
 
+ifeq ($(MEASURED_BOOT_FTPM),y)
+OPTEE_OS_COMMON_EXTRA_FLAGS += EARLY_TA_PATHS=$(OPTEE_FTPM_PATH)/out/bc50d971-d4c9-42c4-82cb-343fb7f37896.stripped.elf
+optee-os-common: ftpm
```

Comment:
Shouldn't `.PHONY: optee-os-common` be set before 1st occurrence of the `optee-os-common` target?

Reply:
It doesn't matter (I tested to create an `optee-os-common` file) and the gnu make documentation doesn't mention it explicitly, but it's good practice. I'll move it.